src/tr/uzaymanga/src/eu/kanade/tachiyomi/extension/tr/uzaymanga/UzayManga.kt

```diff
@@ -58,15 +58,82 @@ class UzayManga : ParsedHttpSource() {
 
-    override fun latestUpdatesSelector() = popularMangaSelector()
+    override fun latestUpdatesSelector() = throw UnsupportedOperationException("Not used")
 
-    override fun latestUpdatesFromElement(element: Element) = popularMangaFromElement(element)
+    override fun latestUpdatesFromElement(element: Element) =
+        throw UnsupportedOperationException("Not used")
 
-    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> {
+        return client.newCall(latestUpdatesRequest(page))
+            .asObservableSuccess()
+            .map { response ->
+                latestUpdatesParse(response)
+            }
+    }
+
+    override fun latestUpdatesParse(response: Response): MangasPage {
+        val document = response.asJsoup()
+
+        // Ana sayfadaki grid içinde bulunan manga kartlarını seçiyoruz
+        val elements =
+            document.select("div.grid.grid-cols-1.lg\\:grid-cols-2.xl\\:grid-cols-3 > div")
+
+        val mangas = mutableListOf<SManga>()
+
+        for (i in 0 until elements.size) {
+            try {
+                val element = elements[i]
+
+                // Manga başlığı
+                val titleElement = element.select("h2")
+                if (titleElement.isEmpty()) continue
+                val title = titleElement[0].text()
+
+                // Manga URL'si
+                val linkElements = element.select("a")
+                if (linkElements.isEmpty()) continue
+                val mangaUrl = linkElements[0].attr("href")
+                if (!mangaUrl.contains("/manga/")) continue
+
+                // Manga kapak resmi
+                val imgElements = element.select("div.card-image img")
+                val thumbnail = if (imgElements.isNotEmpty()) imgElements[0].attr("src") else ""
+
+                // En son bölüm bilgisi
+                val chapterElements = element.select("div.flex.pt-0.flex-col.gap-2\\.5 > a")
+                val hasNewChapter = chapterElements.isNotEmpty()
+
+                if (hasNewChapter) {
+                    val manga = SManga.create().apply {
+                        this.title = title
+                        setUrlWithoutDomain(mangaUrl)
+                        thumbnail_url = when {
+                            thumbnail.startsWith("https://") || thumbnail.startsWith("http://") -> thumbnail
+                            thumbnail.startsWith("//") -> "https:$thumbnail"
+                            thumbnail.isEmpty() -> null
+                            thumbnail.startsWith("/") -> baseUrl + thumbnail
+                            else -> "https://cdn1.uzaymanga.com/$thumbnail"
+                        }
+                    }
+                    mangas.add(manga)
+                }
+            } catch (e: Exception) {
+                // Hata durumunda pas geç
+            }
+        }
```

Comment:
same comments as the other pr,

dont swallow exceptions,
use `selectFirst` and null instead of `isEmpty`
use `forEach` or `map`
use `absUrl("src")` for thumbnail